mock/server.go

```diff
@@ -0,0 +1,31 @@
+package mock
+
+import (
+	"github.com/google/flatbuffers/go"
+	"github.com/soramitsu/iroha-go/iroha"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type mockSumeragiServer struct{}
+
+func (ss mockSumeragiServer) Torii(ctx context.Context, ev *iroha.Transaction) (*flatbuffers.Builder, error) {
+	builder := flatbuffers.NewBuilder(0)
+	builder.Finish(0)
+	// TODO (@upamune): return iroha.Transaction as *flatbuffers.Builder
```

Comment:
should be return this scheme https://github.com/soramitsu/iroha-go/blob/master/schema/endpoint.fbs#L8-L12